index.html

```diff
@@ -154,11 +154,11 @@ <h2>CircuitPython is based on Python</h2>
            srcset="{{ "/assets/images/raspberry_pi.png" | relative_url }} 1x,
                    {{ "/assets/images/raspberry_pi@2x.png" | relative_url }} 2x,
                    {{ "/assets/images/raspberry_pi@3x.png" | relative_url }} 3x"  alt="Raspberry Pi">
-      <h2>CircuitPython libraries on single board computers</h2>
+      <h2>CircuitPython libraries on single-board computers</h2>
```

Comment:
Either spelling is correct, but for the sake of consistency with other documentation, I think this should remain without the hyphen.